common/terminal/colors.go

```diff
@@ -52,6 +57,10 @@ func Warning(s string) string {
 
 // colorize s in color.
 func colorize(s, color string) string {
+	if runtime.GOOS == "windows" {
```

Comment:
This check is superflous. The `IsTerminal` is always called first. We will eventually try to get "some Windows support", so having the check in one place only would make that easier.

Reply:
OK, done.